libs/components/forms/src/lib/modules/file-attachment/file-attachment/file-attachment.component.html

```diff
@@ -41,51 +42,42 @@
     (drop)="fileDrop($event)"
   >
     <div
+      #fileDropDescriptionRef="skyId"
       aria-hidden="true"
       class="sky-screen-reader-only"
       role="tooltip"
-      [attr.id]="fileDropDescriptionElementId"
+      skyId
     >
       {{ 'skyux_file_attachment_file_upload_drag_or_click' | skyLibResources }}
     </div>
     <input
+      #fileInputRef
       hidden
       tabindex="-1"
       type="file"
       [attr.accept]="acceptedTypes || null"
       [disabled]="disabled"
       [required]="isRequired"
       (change)="fileChangeEvent($event)"
-      #fileInput
     />
     <ng-container *ngIf="showFileAttachmentButton">
       <button
         *ngIf="showFileAttachmentButton"
         class="sky-file-attachment-btn sky-btn sky-btn-default"
         type="button"
-        skyId
         [attr.aria-describedby]="
-          (hintText ? hintTextEl.id + ' ' : '') + fileDropDescriptionElementId
-        "
-        [attr.aria-label]="
```

Comment:
I moved the content of `aria-label` to separate `span` elements (see bottom of template) to alleviate the confusion of having both `aria-label` and `aria-labelledby` attributes on the same element.